### PR TITLE
Refactor/rounding issue

### PR DESCRIPTION
**Pull request by detectivekim**
1. `write` function has to write option amount exactly same with input param. same as well for `cancel`, `exercise`, `claim` function
2. Test for call option is not written yet. (test structure will be changed.)

### CODE REVIEW

File: test/mocks/MockCallOptionToken.sol [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+// License: https://license.clober.io/LICENSE.pdf
+
+pragma solidity ^0.8.0;
+
+import "../../contracts/CallOptionToken.sol";
+
+contract MockCallOptionToken is CallOptionToken {
+    constructor(
+        address quoteToken,
+        address underlyingToken,
```
Comment (JhChoy) [inline test/mocks/MockCallOptionToken.sol]:
```suggestion
        address underlyingToken,
        address quoteToken,
```

File: test/mocks/MockCallOptionToken.sol [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        uint256 strikePrice,
+        uint256 expiresAt,
+        uint256 exerciseFee
+    ) CallOptionToken(quoteToken, underlyingToken, strikePrice, expiresAt, exerciseFee, "Mock Call Option", "M-C") {}
```
Comment (JhChoy) [inline test/mocks/MockCallOptionToken.sol]:
```suggestion
    ) CallOptionToken(underlyingToken, quoteToken, strikePrice, expiresAt, exerciseFee, "Mock Call Option", "M-C") {}
```

Comment (JhChoy) [reply test/mocks/MockCallOptionToken.sol]: `MockPutOptionToken` also has the same issue.

File: test/mocks/MockCallOptionToken.sol (hunk 1 above)
Comment (detectivekim) [reply test/mocks/MockCallOptionToken.sol]: solved.

File: test/mocks/MockCallOptionToken.sol (hunk 2 above)
Comment (detectivekim) [reply test/mocks/MockCallOptionToken.sol]: solved.

**Review by JhChoy — APPROVED** [review]

**Pull request merged**